src/index.js

```diff
@@ -6,18 +6,22 @@ import { BrowserRouter, Route } from 'react-router-dom';
 
 import './styles/index.css';
 import reducer from './reducers';
+import Header from './components/Header';
 import App from './containers/App';
 import Event from './containers/Event';
+import Footer from './components/Footer';
 import Confirmation from './containers/Confirmation';
 
 const store = createStore(reducer);
 ReactDOM.render(
   <Provider store={store}>
     <BrowserRouter>
-      <div id="router">
+      <div id="router" className="flex column">
```

Comment:
@bill-feng Could you explain what this `flex column` is for?

Reply:
- flex is short hand for setting a component's display to flex.
- column is short hand for setting flex direction to column.

Reply:
@bill-feng I mean why you need to set flex and column for #router. I'll ask you in person later :)

Reply:
It's so that the items in the div gets drawn vertically, and on top of that we can have flex box properties to use. eg. adding flex-grow to the center components so that footer gets pushed to the bottom.